pootle/apps/pootle_store/views.py

```diff
@@ -415,6 +415,7 @@ def get_alt_srcs(self):
     def get_queryset(self):
         return Unit.objects.get_translatable(self.request.user).select_related(
             "store",
+            "store__filetype__name",
```

Comment:
yep for select relateds only the table parts is significant - altho the other part doesnt break anything